src/main/java/pl/pjwstk/woloappapi/security/oauth2/OAuth2UserInfoFactory.java

```diff
@@ -0,0 +1,15 @@
+package pl.pjwstk.woloappapi.security.oauth2;
+
+import pl.pjwstk.woloappapi.security.AuthProvider;
+
+import java.util.Map;
+
+public class OAuth2UserInfoFactory {
+    public static OAuth2UserInfo getOAuth2UserInfo(String registrationId, Map<String, Object> attributes) {
+        if(registrationId.equalsIgnoreCase(AuthProvider.google.toString())) {
+            return new GoogleOAuth2UserInfo(attributes);
+        } else {
```

Comment:
`else` niepotrzebny.

Kod łatwiej przeczytać, gdy jest taki:

```
public static OAuth2UserInfo getOAuth2UserInfo(String registrationId, Map<String, Object> attributes) {
    if(registrationId.equalsIgnoreCase(AuthProvider.google.toString())) {
        return new GoogleOAuth2UserInfo(attributes);
    }

    throw new OAuth2AuthenticationProcessingException("Sorry! Login with " + registrationId + " is not supported yet.");
}
```

`return` kończy wykonanie tej metody, więc else nie będzie wykonany.

Jeśli w `if` jest `false` to `if` zostanie ominięte, a wykonany dalszy kod. 

Stosowanie `else` niepotrzebnie zwiększa zawiłość kodu (w tym np. w sonarze wskazywałby wyższy poziom złożoności).